daemon/cmd/daemon_main.go

```diff
@@ -1122,6 +1122,12 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.MarkHidden(option.MaxInternalTimerDelay)
 	option.BindEnv(vp, option.MaxInternalTimerDelay)
 
+	flags.Bool(option.EnablePerNodeLabels, defaults.EnablePerNodeLabels, "Enable use of node label based identity")
```

Comment:
I had wondered (thread on Slack [here](https://cilium.slack.com/archives/C2B917YHE/p1705648044575499)) whether this should be something like `--enable-per-node-policy` or similar since k8s already has per-node labels and Cilium doesn't control it. But I don't feel overly strongly about this.

Reply:
Nate suggested `enable-node-selector-labels`, which I also think is a bit more precise.

Reply:
changed to `enable-node-selector-labels` :)